README.md

```diff
@@ -1,2 +1,48 @@
 # brew-buddy-coffee-companion
 Companion app for tracking daily brewing and logging data
+
+
+## Routes
+All routes are found at the `/api/` endpoint.
+
+### Origin
+A coffee origin includes where the coffee comes from, and the recommended brew method. All routes require an authorized user.
+
+* `POST -- /origin` - Creates a new origin.
```

Comment:
Even though you specified @ the top that all routes are found at `/api` is better to list all your routes as there absolute path 

eg: `/api/origin`